### DISCUSSION TITLE
LaTeX equation not rendering in <figcaption>

### DISCUSSION THREAD

**Discussion by broktoram** [squidfunk/mkdocs-material#5516] [General]
I'm using the md_in_html extension to create and centre a figure with a caption and would like to include a LaTeX equation as the caption. 

However, neither $ nor $$ seems to work in the <figcaption> element and I only see the raw LaTeX rather than the rendered version.

Is it possible to have a LaTeX equation as the caption of a figure?

**Comment by broktoram** [comment]
Update:
I was able to get this to work by including a span element with the arithmatex class around the equation in the figcaption.

**Reply by facelessuser** [reply]
If you are using `pymdownx.arithmatex`, there is nothing that prevents you from using LaTeX math in figcaptions like so:

```html
<figcaption markdown>
$3 + 3$
</figcaption>
```

You can see it working below:

<img width="448" alt="Screenshot 2023-05-15 at 7 36 16 AM" src="https://github.com/squidfunk/mkdocs-material/assets/1055125/595a963f-e50e-4e0f-aead-f0d5583e0fb2">
